tests/testthat/test-plot.ppi.R

```diff
@@ -2,29 +2,31 @@ data(example_scan)
 ppi <- project_as_ppi(example_scan)
 example.plot <-plot(ppi, param = "VRADH", zlim = c(-5, 5))
 
-test_that("plot.ppi() returns error on incorrect parameters", {
+test_that("plot() returns error on incorrect parameters", {
+  # use plot.ppi() to avoid defaulting to base plot()
   expect_error(plot.ppi("not_a_ppi"))
-  expect_error(plot.ppi(ppi, param = "not_a_param"))
-  expect_error(plot.ppi(ppi, quantity = "DBZH"))
+  expect_error(plot(ppi, param = "not_a_param"))
+  expect_error(plot(ppi, quantity = "DBZH"))
 })
 
 test_that("if param missing default to DBZH", {
-  expect_equal(plot.ppi(ppi), plot.ppi(ppi, param = "DBZH"))
+  expect_equal(plot(ppi), plot(ppi, param = "DBZH"))
 })
 
-test_that("plot.ppi() returns ggplot object", {
-  expect_true(all(class(plot.ppi(ppi)) == c("gg", "ggplot")))
+test_that("plot() returns ggplot object", {
+  expect_true(all(class(plot(ppi)) == c("gg", "ggplot")))
```

Comment:
```suggestion
  expect_s3_class(plot(ppi), c("gg", "ggplot"))
```

Please use `expect_s3_class()` for class validation